chazz/__init__.py

```diff
@@ -192,22 +209,22 @@ def get_running_instance(config):
         return get_instance(config.ec2, inst['InstanceId'])
 
 
-def ssh_host(config, host):
+def ssh_host(config, host, username):
```

Comment:
Instead of threading through an additional `username` parameter, let's make `username` a field of the `config` object that's already passed around to everything.